src/Surfnet/Stepup/Identity/Entity/VerifiedSecondFactor.php

```diff
@@ -159,6 +144,16 @@ public function vet($registrationCode, $secondFactorIdentifier, $documentNumber,
         );
     }
 
+    public function hasEqualOrHigherLoaComparedTo(SecondFactor $comparable)
```

Comment:
repeated three times in the exact same manner, smells like abstraction could be done?